KnapSack.py

```diff
@@ -0,0 +1,27 @@
+"""
+Time complexity 2^n
+Space Complexity O(1)
```

Comment:
Wouldn't the space be O(n + capacity) considering recursive stack

Reply:
Yeah right. I missed it. Thanks